package.json

```diff
@@ -16,7 +16,7 @@
     "test:playWright": "npx playwright test",
     "test:playWrightWithUpdate": "npx playwright test --update-snapshots",
     "coverage": "vitest run --coverage",
-    "precommit": "concurrently \"npx lint-staged\" \"npm:build\"",
+    "precommit": "concurrently \"chmod +x .husky/prepare-commit-msg\" \"npx lint-staged\" \"npm:build\"",
```

Comment:
Can we make `prepare-commit-msg` to run first and if it success, then run `lint` and `buld`

Reply:
Done